packages/atlas/src/views/viewer/VideoView/CommentsSection.tsx

```diff
@@ -89,10 +91,29 @@ export const CommentsSection: React.FC<CommentsSectionProps> = ({ disabled, vide
     commentInputProcessing,
     deleteComment,
     moderateComment,
+    updateComment,
   } = useReactionTransactions()
 
   const mdMatch = useMediaMatch('md')
-  const placeholderItems = loading && !comments ? Array.from({ length: 4 }, () => ({ id: undefined })) : []
+
+  const setCommentBody = ({ commentId, comment }: { commentId: string; comment: string | undefined }) =>
```

Comment:
I'm not a big fan of creating variables like `_setSomething` and `setSomething`. Couldn't we rename this handler handleCommentBodyChange or smth?